lib/active_shipping/address_standardization_response.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveShipping
+
+  # The `AddressStandarizationResponse` object is returned by the {ActiveShipping::Carrier#????}
+  # call. The most important method is {#standardize_addresses}, which will return a list of 
+  # standardized, correctly-spelled, validated, full addresses.
+  #
+  # @note This only works for USPS and in the US
+  #
+  # @!attribute addresses
+  #    The correct addresses returned from USPS
+  class AddressStandardizationResponse < Response
+
+  	attr_reader :locations
+
+  	# Initializes a new AddressStandarizationResponse instance.
+    #
+    # @param success (see ActiveShipping::Response#initialize)
+    # @param message (see ActiveShipping::Response#initialize)
+    # @param params (see ActiveShipping::Response#initialize)
+    # @option options (see ActiveShipping::Response#initialize)
```

Comment:
Could summarize it in one line like https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/tracking_response.rb#L59